Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -229,7 +235,11 @@ - (NSDictionary *)liveContext
     });
 
     self.location = !self.location ? [self.configuration shouldUseLocationServices] ? [SEGLocation new] : nil : self.location;
+
+#if TARGET_OS_IOS || (TARGET_OS_MAC && !TARGET_OS_IPHONE)
     [self.location startUpdatingLocation];
+#endif
```

Comment:
I don't know that I agree. CoreLocation is still available on tvOS, as well as CLLocationManager, but it doesn't provide periodic updates like it does on iOS. Presumably, this is because Apple doesn't anticipate the device moving while it's in use. 😜